content/millennium/r4/individuals/practitioner.md

```diff
@@ -89,6 +89,58 @@ List an individual Practitioner by its id:
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Create
+
+Create an individual Practitioner.
+
+    POST /Practitioner
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', 'Content-Type': 'application/fhir+json'} %>
```

Comment:
should the gt here be &gt;?

Reply:
https://github.com/cerner/fhir.cerner.com/pull/330/commits/c128b759084993b859b499ef4e9e83aedf8a4898#diff-00a306a2aa1334ff6f799b4200c75030R104

Reply:
to my knowledge it does not have to be